### DISCUSSION TITLE
When will Copilot X support running on IntelliJ IDEA?

### DISCUSSION THREAD

**Discussion by luoxiaofeng1029** [community/community#56175] [Copilot Conversations]
### Select Topic Area

Question

### Body

Currently, it seems to only support running on VS Code.

**Comment by Akash1134** [comment]
Hi @luoxiaofeng1029 ,

GitHub Copilot X is currently a representation of GitHub’s vision for the future rather than an available product offering of GitHub Copilot 

At this time, Copilot X is supported on VS or VSCode users but we're taking the time to determine the best way to provide IDE support to our customers. 

Thanks!

**Reply by shayneoneill** [reply]
Please do. Its a great product, but its a competitive space right now (with most of the others still back ending off code-davinci-??? anyway, and it would be *really* good to be able to just ask questions about a codebase.

**Comment by funaiy** [comment]
@Akash1134  Is there a foreseeable timeline to support idea? We are all looking forward to the landing of CopilotX in IntelliJ IDEA.

**Comment by allanlaal** [comment]
any progress? 

Don't prove the ppl right that said Github is gonna start pushing M$ products after the acquisition

**Comment by d4forevr** [comment]
Would love an update. Looking forward to be using Copilot X but I'm way too locked into the Jetbrains toolset. Please don't ignore us..

**Comment by dJani97** [comment]
Also waiting for this.

**Comment by morovinger** [comment]
same here

**Comment by Nincha** [comment]
Adding my humble +1. :]

**Comment by Jozaguts** [comment]
another one ++

**Comment by truethari** [comment]
Any updates?

**Reply by keyboardSmasher** [reply]
Waiting as well.

**Comment by Imccccc** [comment]
any updates?

**Comment by ObaidaAlhaasan** [comment]
Any updates?

**Comment by Maqsim** [comment]
+++

**Comment by mdsadiqueinam** [comment]
any decision

**Comment by DevSock** [comment]
I'm sorry to be so presumptive, but it really feels like Copilot X support hasn't been introduced to IntelliJ because of the nature of it being directly competitive with IntelliJ's recently released "AI Assistant". I really hope this isn't the case, but it does feel like so. Given Copilot X has been available on VSC for going on 5 months, it seems odd that it isn't supported in some fashion within IntelliJ. 

I also may just be reading into things further than I should, but the overlap between availability of both products does seem conflicting in nature. I do hope that I am just feeling this way and it isn't actually the case, but it also doesn't help that I've been on the wait-list for the AI Assistant for almost a month at this point, while I've had access to Copilot X since the day it released, so I'm feeling slightly stifled in that department.

Here's hoping developers aren't being intentionally stifled via either company at hand, due to whatever conflict there may be between the services. That would truthfully leave a sour taste in the mouths of many I feel.

**Reply by Autuan** [reply]
JetBrains' AI assistant requires EAP version or 2023.2 (with a plugin installed) currently.
I find it too challenging. It did not support VPN proxy in the previous version. :( Therefore, I stopped using it.
JetBrains' toolset is essential for my development and AI assistant like Copilot X is also very helpful.

I wish Copilot X would be available on JetBrains soon.

Besides, AI assistant and Copilot have some similar features. I suppose most people would only pay for one of them, not both.

**Comment by maxfahl** [comment]
Any news on the matter?

**Comment by VinGits** [comment]
![image](https://github.com/community/community/assets/142453346/5e0056ee-4f90-4ab4-8573-4c66f0c9211b)
false advertisement?

**Reply by raducretu** [reply]
+1 Jetbrains

**Comment by philiparpin** [comment]
+1

**Comment by andergsx** [comment]
+1

**Comment by keyboardSmasher** [comment]
The silence is deafening.

**Comment by iamsyh** [comment]
+1

**Comment by glenncai** [comment]
push!!!

**Comment by tiredpillow** [comment]
+1

**Comment by aananiev** [comment]
+1 ^^

**Comment by keviocastro** [comment]
+1

**Comment by mxvsh** [comment]
when ++

**Comment by sunday-floris-tisseyre** [comment]
+1

**Comment by zorlac** [comment]
+1 we are paying for the feature that is only available for VS code users?

**Comment by MarkvanOsch** [comment]
Also stating the obvious here: +1

**Comment by maxfahl** [comment]
I don't know if you guys are aware, since you keep asking here, but they did announce that Copilot Chat is coming to Jetbrains IDE's this month.

**Reply by nicmar** [reply]
So is that the complete feature set that is in VS? There’s a private beta that you can sign up for, not sure how long the wait is.

**Comment by keyboardSmasher** [comment]
Jetbrains AI is a flop in it's current state. 
"I'm sorry, but as an AI, I don't have direct access to your codebase or database."

What good are you then? I already have a ChatGPT Plus account.
